packages/ckeditor5-list/src/listproperties/listpropertiesui.ts

```diff
@@ -426,14 +446,16 @@ function getMenuBarStylesMenuCreator(
 		parentCommandName,
 		buttonLabel,
 		styleGridAriaLabel,
-		styleDefinitions
+		styleDefinitions,
+		isStyleVisible
 	}: {
 		editor: Editor;
 		propertiesConfig: Readonly<ListPropertiesConfig>;
 		parentCommandName: 'bulletedList' | 'numberedList';
 		buttonLabel: string;
 		styleGridAriaLabel: string;
 		styleDefinitions: Array<StyleDefinition>;
+		isStyleVisible: boolean;
```

Comment:
This won't be needed if you pass a normalized config in the `propertiesConfig` parameter.